### PR TITLE
odim:4875 : Added attribute capacity GB for drives

### PR DESCRIPTION
**Pull request by skhan-anjum**
Adding capacityGB in struct for iloplugin to calculate capacitybytes from capacityGB instead of capacityMiB

### CODE REVIEW

**Review by shivacharanms — APPROVED** [review]

File: lib-dmtf/model/storage.go [hunk 1]
```diff
@@ -158,6 +158,7 @@ type Drive struct {
 	AssetTag                      string             `json:"AssetTag,omitempty"`
 	BlockSizeBytes                int                `json:"BlockSizeBytes,omitempty"`
 	CapableSpeedGbs               int                `json:"CapableSpeedGbs,omitempty"`
+	CapacityGB                    int                `json:"CapacityGB,omitempty"`
```
Comment (bharath-b-hpe) [inline lib-dmtf/model/storage.go:161]: CapacityGB is not part of standard Drive schema, please revert this change
